pkg/cli/cmds/executions.go

```diff
@@ -3,8 +3,9 @@ package cmds
 import (
 	"encoding/base64"
 	"fmt"
+	"time"
 
-	"github.com/rancher/terraform-controller/pkg/age"
+	"github.com/docker/go-units"
```

Comment:
Is this used?

Reply:
yup in the cli execution list to show when the execution was created

Reply:
It should have an alias name then, use goimports to format